### PR TITLE
buttons and more numerical input options

### PR DESCRIPTION
**Pull request by Ericxu19**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,13 +68,38 @@
 
 let population = parseInt(document.getElementById("population").value);
 let previous_population = parseInt(document.getElementById("population").value);
+let previous_initial_rate = parseInt(document.getElementById("initialInfectionRate").value);
```
Comment (zengyifa) [inline index.html]: do you need these new variables or can you not just use the current population and initial infection rate?

**Review by zengyifa — CHANGES_REQUESTED** [review]
I think +- 1 is too small for population which has a huge range. maybe something like +- 100 is more fitting. 

The buttons seem to have different borders on my screen, is that intentional? I'm using chrome on windows.

I'm not sure green is the best background, it blends in too well, we want to make it stand out. Maybe blue or no background works.

File: index.html (hunk 1 above)
Comment (zengyifa) [reply index.html]: generally try to use what's already there.

Comment (Ericxu19) [reply index.html]: This was used when I first try to update the values as soon as the user has typed in a number. This is indeed no longer needed, I will change that, thanks.

**Pull request closed without merging**